pkg/startup/startup.go

```diff
@@ -720,6 +739,55 @@ func configureIPPools(ctx context.Context, client client.Interface) {
 		return
 	}
 
+	// If CIDRs weren't specified through the environment variables, check if they're present in kubeadm's
+	// config map.
+	if clientset != nil && (len(ipv4Pool) == 0 || len(ipv6Pool) == 0) {
+		kubeadmConfig, err := clientset.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(KubeadmConfigConfigMap, metav1.GetOptions{})
```

Comment:
Should we use the `queryKubeadmConfigMap` here?

Reply:
With the code review changes from earlier today `queryKubeadmConfigMap` became unnecessary. It's cleaner to call `Get()` directly on both use cases.